File: ruby/car.rb
class Car
    @@honkCount

    def initialize
        @@honkCount = 0
    end

    def honk
        @@honkCount += 1

        "You honked #{@@honkCount} times"
    end

    def honkMultipleTimes(num)
        @@honkCount += num

        'You honked up to ' + num.to_s + ' more times bringing your total to ' + @@honkCount.to_s
    end
end

File: python/car.py
class Car:
    __honkCount = 0

    def __init__(self):
        self.__honkCount = 0
    
    def honk(self):
        self.__honkCount += 1

        return "You honked " + str(self.__honkCount) + " times"

    def honkMultipleTimes(self, num):
        self.__honkCount += num

        return "You honked up to " + str(num) + " more times bringing your total to " + str(self.__honkCount)